budgetportal/webflow/urls.py

```diff
@@ -12,4 +12,9 @@
         views.provincial_infrastructure_project_detail,
         name="provincial-infra-project-detail",
     ),
+    url(
+        r"^api/v1/infrastructure-projects/provincial/search/$",
```

Comment:
```suggestion
        r"^api/v1/infrastructure-projects/provincial/$",
```

Reply:
we can add to this API later, e.g. `api/v1/infrastructure-projects/provincial/123` returning project id 123

Reply:
this is a REST API meaning the URL is the identifier for the _resource_. the resource is the _collection of projects_

_search_ isn't part of the identity of the resource. Search is just an operation you can do with the resource using the HTTP GET method.

Reply:
Not saying REST is perfect, but while we can follow conventions, it's good to :)